src/worker/python/schema_migration.py

```diff
@@ -19,23 +19,29 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-"""Module defining methods used in schema migration of Worker database.
+"""Module defining methods used in schema migration of Worker metadata database.
```

Comment:
I think we figured out that we need wdb or worker schema management, but not both, right? Can one of them be removed? (Or, why make the changes to wdb and how are they tested?)

Reply:
Well, that's is one of the questions I too wanted to ask you. I'm leaning toward removing the `worker` module entirely in favor of the existing module `wdb` because the latter has C++ classes depending on the database schema. And the former (f I understood you correctly) was introduced purely for schema management purposes.
I can do this if you agree (?)

Reply:
yes, I agree.

Reply:
This will be postponed before we'll decided on a comprehensive solution for schema management (which I expect will also affect the schema migration tools).